test/medium/fb_messenger/fb_messenger_test.exs

```diff
@@ -0,0 +1,14 @@
+defmodule Medium.FbMessengerTest do
+  use ExUnit.Case
+
+  alias Medium.FbMessenger
+
+  describe "process_messages/1" do
```

Comment:
There's no enough documentation about how should be the proper parameters for these functions.
e.g. `FbMessenger.process_messages([%{"messaging"=> %{?}}])`